sstables/sstables.cc

```diff
@@ -573,7 +573,7 @@ future<> parse(sstable_version_types v, random_access_reader& in, summary& s) {
                         auto position = seastar::read_le<uint64_t>(buf.get());
                         auto token = dht::global_partitioner().get_token(key_view(key_data));
                         s.add_summary_data(token.data());
-                        s.entries.push_back({ dht::token(dht::token::kind::key, std::move(token._data)), key_data, position });
+                        s.entries.push_back({ token, key_data, position });
```

Comment:
Why this change? It makes sense, but is unrelated to the patch, no?

Reply:
I moved it out to a separate patch.